lib/com.c

```diff
@@ -66,7 +66,18 @@
 #  endif
 #  define MSG_NOSIGNAL 0
 #endif /* USE_MSG_NOSIGNAL */
+
+static int64_t queue_count = 0;
```

Comment:
We should count only the size of the target queue. We should not count all size.